Converters/Autocad/Speckle.Converters.AutocadShared/ToSpeckle/Geometry/HatchToSpeckleConverter.cs

```diff
@@ -112,8 +106,93 @@ private ADB.Curve PolylineFromLoop(ADB.HatchLoop loop)
         }
         return polyline;
       }
+
+      // if .Polyline is null, read from .Curves
+      if (loop.Curves.Count == 0)
+      {
+        throw new ConversionException($"Hatch loop doesn't contain any segments.");
+      }
+
+      if (loop.Curves.Count > 1) // handle the multi-segment case only with Line segments (not able to produce a Hatch Loop with multiple segments of other types)
+      {
+        foreach (var lineSegment in loop.Curves)
```

Comment:
this should be converted to Line entities right? why do you need to convert it to Polyline?